<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExposeConfirmationDialog</class>
 <widget class="QDialog" name="ExposeConfirmationDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Внимание!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblBegDate">
       <property name="text">
        <string>За период с</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBegDate"/>
     </item>
     <item>
      <widget class="QLabel" name="lblEndDate">
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDate"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkFilterPaymentByOrgStructure">
     <property name="text">
      <string>При выставлении счетов учитывать текущее подразделение</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkReExpose">
     <property name="text">
      <string>Выполнять перевыставление по имеющимся отказам</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkSeparateReExpose">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Перевыставлять в отдельный счет</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkMesCheck">
     <property name="text">
      <string>Проверять на соответствие МЭС</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="chkOnlyDispCOVID">
     <property name="text">
      <string>Выставлять только углубленную диспансеризацию</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="chkOnlyResearchOnCOVID">
     <property name="text">
      <string>Выставлять только исследования на covid</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblMessage">
         <property name="text">
          <string>message</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkFilterPaymentByOrgStructure</tabstop>
  <tabstop>chkReExpose</tabstop>
  <tabstop>chkSeparateReExpose</tabstop>
  <tabstop>chkMesCheck</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExposeConfirmationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExposeConfirmationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkReExpose</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkSeparateReExpose</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
